modal/_functions.py

```diff
@@ -785,6 +785,16 @@ def from_local(
             scaledown_window=scaledown_window,
         )
 
+        # For clustered functions, container settings must be multiples of cluster_size
+        if cluster_size is not None and cluster_size > 1:
```

Comment:
I know I suggested this in the doc, but I think this is a breaking change. For me, I'm okay with it breaking.

Do we have users that set `*_containers` to a value that is not a multiple of `cluster_size`?